frontend/src/metabase/dashboard/hooks/use-embed-theme.ts

```diff
@@ -7,7 +7,9 @@ import type { DisplayTheme } from "metabase/public/lib/types";
 
 import type { EmbedThemeControls } from "../types";
 
-export const useEmbedTheme = (): EmbedThemeControls => {
+export const useEmbedTheme = (
```

Comment:
If a user passes a theme, then we want to use their theme instead of our default of `light`